Added prerequisites for remote services in node (#1628)
cap-js/docs: guides/using-services.md

It should be also possible to search for suppliers and show the associated risks by extending the remote supplier service [with the local risk service](#extend-a-remote-by-a-local-service) and its risks.

> ## Prerequisites { .node }

Reviewer: ```suggestion
## Install Dependencies { .node }
```

> 
> First install the required packages:
> 

Reviewer: ```suggestion
```

To prevent accidental loss of modifications, the `cds import --as cds` command refuses to overwrite modified files based on a "checksum" that is included in the file.

### Install Dependencies { .node }

> In the next section, you're going to mock a remote service as OData service. For this you need to install the required packages first:

Author: @renejeglinsky These dependencies are needed not only for the mock scenario,  they are needed for all remote service scenarios.

Reviewer: ```suggestion
In the next section, you're going to mock a remote service as OData service. To make this and all other remote service interaction possible, you need to install the required packages first:
```

Reviewer 2: It's independent of mocking, you always need these packages.

Reviewer: The challenge is to find the best place for this step. Usually (grow as you go) we make the user execute such things when they first need it. In this guide, if you go from top to bottom, that would be this place, right? If we make it a step like a prerequisite, you'd install the dependencies and then? Why do it if you not need it immediately? Anyway the referencing part is much more important so that people jumping into the guide and not reading from top to bottom can find that information as well.

Reviewer 2: Installing the dependencies is very important, otherwise you cannot consume remote services. If we bury it within some minor step, people will overlook it and create issues on our side (happened before many times).
